doc-coverage/info.rkt

```diff
@@ -1,4 +1,6 @@
 #lang info
 
-(define name "doc-coverage")
-(define scribblings '(("doc-coverage.scrbl" ())))
\ No newline at end of file
+(define name "docs-coverage")
+(define scribblings '(("doc-coverage.scrbl" ())))
+(define raco-commands
+  '(("cover-doc" (submod doc-coverage/raco main) "a code documentation coverage tool" 25)))
```

Comment:
`"doc-coverage"` is probably a better command name, so as to match the library name. Arguably the entire library should be named "cover-doc", but that's a somewhat bigger change